docs/python/latest/modules/keanu/vertex/generated.html

```diff
@@ -160,7 +160,7 @@ <h1>Source code for keanu.vertex.generated</h1><div class="highlight"><pre>
 <span class="n">java_import</span><span class="p">(</span><span class="n">context</span><span class="o">.</span><span class="n">jvm_view</span><span class="p">(),</span> <span class="s2">&quot;io.improbable.keanu.vertices.dbl.nonprobabilistic.operators.binary.MultiplicationVertex&quot;</span><span class="p">)</span>
 <span class="n">java_import</span><span class="p">(</span><span class="n">context</span><span class="o">.</span><span class="n">jvm_view</span><span class="p">(),</span> <span class="s2">&quot;io.improbable.keanu.vertices.dbl.nonprobabilistic.operators.binary.PowerVertex&quot;</span><span class="p">)</span>
 <span class="n">java_import</span><span class="p">(</span><span class="n">context</span><span class="o">.</span><span class="n">jvm_view</span><span class="p">(),</span> <span class="s2">&quot;io.improbable.keanu.vertices.dbl.nonprobabilistic.operators.multiple.ConcatenationVertex&quot;</span><span class="p">)</span>
-<span class="n">java_import</span><span class="p">(</span><span class="n">context</span><span class="o">.</span><span class="n">jvm_view</span><span class="p">(),</span> <span class="s2">&quot;io.improbable.keanu.vertices.dbl.nonprobabilistic.operators.multiple.DoubleSetWithMaskVertex&quot;</span><span class="p">)</span>
+<span class="n">java_import</span><span class="p">(</span><span class="n">context</span><span class="o">.</span><span class="n">jvm_view</span><span class="p">(),</span> <span class="s2">&quot;io.improbable.keanu.vertices.dbl.nonprobabilistic.operators.ternary.DoubleSetWithMaskVertex&quot;</span><span class="p">)</span>
```

Comment:
This is annoying, seems the docs have become out of date again. Going to raise a ticket for adding a step to check the docs are up to date in buildkite before allowing merge.